tripleblindmarket/private-kit
# Hotfix 1.0.65 copy & legal changes -> Develop

#### Description:

Adds the copy changes from master (1.0.65) branch to develop so we can unblock BT team. 
Changes:
- legal screen 
- HA selection screen 
- on-boarding flow (removes `Onboarding4`)
- `.en` and `.ht` string files

> NOTE: this defaults strings to `1.0.65`'s version, except for the string `label.covid_safe_paths` on the legal screen, which was added in develop branch. 

I also did some cleanup on the navigation `entry.js` & onboarding screens while touching those.

On the legal screen, I added a safe area context to the bottom button (seems like an important button to ensure is clickable across devices).

@efalkner lmk if we need a version bump on this

#### Linked issues:
https://pathcheck.atlassian.net/browse/SAF-406

#### Screenshots:

<img width="570" alt="image" src="https://user-images.githubusercontent.com/25315679/82387952-51bc5600-9a06-11ea-852c-53a8a32cadeb.png">
<img width="570" alt="image" src="https://user-images.githubusercontent.com/25315679/82387931-4bc67500-9a06-11ea-81e4-02304beab86e.png">


#### How to test:

We need to ensure that the updated strings are good to go -- these default to the hotfix version, so if develop has updated any, those will be lost. 

Generally, manual test over the onboarding flow, legal page, and HA page. 

Ensure string populate in Haitian Creole also.
